t/dispatch.t

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env perl
+
+use 5.028;
+use warnings;
+use experimental qw(signatures);
+
+use Test::Lib;
+use Test::PromAlertProxy;
+use Test::More;
+use Test::Time time => Test::PromAlertProxy->now;
+
+use IO::Async::Loop;
+
+use PromAlertProxy::Hub;
+use PromAlertProxy::PromAlert;
+
+use PromAlertProxy::Logger '$Logger' => { init => {
+  ident    => 'promalertproxy',
+  to_self  => 1,
+} };
+
+my $loop = IO::Async::Loop->new;
+
+my $hub = PromAlertProxy::Hub->new(loop => $loop);
+
+my $target1 = Test::PromAlertProxy::Target->new(
+  hub     => $hub,
+  id      => 'test1',
+  default => 1,
+);
+my $target2 = Test::PromAlertProxy::Target->new(
+  hub     => $hub,
+  id      => 'test2',
+);
+my $crashtarget = Test::PromAlertProxy::CrashTarget->new(
+  hub => $hub,
+  id  => 'crashtest',
+);
+$hub->add_target($target1);
+$hub->add_target($target2);
+$hub->add_target($crashtarget);
+
+{
```

Comment:
I'd write these all as subtests, but I'm a weirdo.

Reply:
I should write more tests then probably, form some better habits.